draft-ietf-quic-recovery.md

```diff
@@ -389,6 +398,13 @@ rttvar_sample = abs(smoothed_rtt - adjusted_rtt)
 rttvar = 3/4 * rttvar + 1/4 * rttvar_sample
 ~~~
 
+An endpoint might postpone the processing of acknowledgements when the
```

Comment:
I agree this would be better above.  Also, I think we should make this tighter, since it only applies ACKs of 0-RTT and 0.5-RTT packets.